### ISSUE TITLE
Capture Only Office user interaction events

### ISSUE THREAD

**Issue by ar27111994** [ONLYOFFICE/DocumentServer#556]
**Do you want to request a *feature* or report a *bug*?**
Question
**What is the current behavior?**
Don't know
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
N/A
**What is the expected behavior?**
Suppose our application automatically logs out idle users after some specified amount of time. Is there any way to capture different types of user interaction events from Only Office cross-origin iframe? I see mouse events callback in javascript API but that isn't enough for this scenario.
**Did this work in previous versions of DocumentServer?**
Don't know
**DocumentServer version:**
5.2.8
**Operating System:**
Cent OS
**Browser version:**
71.0.3578.80

**Comment by flaminestone** [comment]
Hello @ar27111994. Do you want to detect an event when the user is idle? Or something else?

**Comment by ar27111994** [comment]
no, I want to detect user events like keydown, mouseup, scroll, touch etc.

**Comment by flaminestone** [comment]
Sorry, we don't plan to add this feature

**Comment by regent112** [comment]
Hello @ar27111994. A bug #80368 has been logged based on your request.
